compiler/rustc_codegen_llvm/src/asm.rs

```diff
@@ -671,6 +672,8 @@ fn reg_to_llvm(reg: InlineAsmRegOrRegClass, layout: Option<&TyAndLayout<'_>>) ->
             InlineAsmRegClass::S390x(S390xInlineAsmRegClass::reg) => "r",
             InlineAsmRegClass::S390x(S390xInlineAsmRegClass::freg) => "f",
             InlineAsmRegClass::Msp430(Msp430InlineAsmRegClass::reg) => "r",
+            InlineAsmRegClass::M68k(M68kInlineAsmRegClass::reg) => "r",
+            InlineAsmRegClass::M68k(M68kInlineAsmRegClass::reg_ptr) => "m",
```

Comment:
The LLVM constraint code for address registers is `a`.

Reply:
Ah. The 68k specific constraints aren't listed in https://llvm.org/docs/LangRef.html#supported-constraint-code-list, which is what I was looking at, but I see [GCC's documentation](https://gcc.gnu.org/onlinedocs/gcc/Machine-Constraints.html) lists this.